library/psa_crypto_core.h

```diff
@@ -212,6 +214,15 @@ psa_status_t psa_copy_key_material_into_slot( psa_key_slot_t *slot,
  */
 psa_status_t mbedtls_to_psa_error( int ret );
 
+/** Get Mbed TLS MD information of a hash algorithm given its PSA identifier
+ *
+ * \param[in] alg  PSA hash algorithm identifier
+ *
+ * \return  The Mbed TLS MD information of the hash algorithm. \c NULL if the
+ *          PSA hash algorithm is not supported.
+ */
+const mbedtls_md_info_t *mbedtls_md_info_from_psa( psa_algorithm_t alg );
```

Comment:
Should this be guarded in a `PSA_BUILTIN_HASH`-like condition? (ref #4157)
Since mbedtls_md_info_t is not part of PSA?

Reply:
I think it is fair to say the `mbedtls_md_info_t` is part of the Mbed TLS PSA implementation. The definition of `mbedtls_md_info_t` is not guarded thus it seems to be ok.